server/database/queries/project/addProject.js

```diff
@@ -0,0 +1,17 @@
+const connection = require('../../config/connection');
+
+const addProject = (
+  name,
+  description,
+  imgUrl,
+  githubLink,
+  websiteLink,
+  projectType,
+  cohortId,
+) =>
```

Comment:
And here just pass `data` which is `req.body` from the previous comment and edit the array below at line 14 to became `[data.name, data.description, data.imgUrl, data.githubLink, data.websiteLink, data.projectType, data.cohortId]`.

Reply:
yes :100: